### PR TITLE
docs: use variables for versions

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

as the title

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/.vitepress/config/plugins.ts [hunk 1]
```diff
@@ -6,18 +6,16 @@ export const replaceVariate = md => {
...
+      if (token) {
+        token.content = token.content.replace(new RegExp(/<%\s*(.*?)\s*%>/, 'g'), (_, $1) => {
+          if (variates[$1]) return `${variates[$1]}`
+          else {
+            return `${_}`
+          }
+        })
       }
-    })
+    }
```
Comment (nicecui) [inline docs/.vitepress/config/plugins.ts:19]: @sunchanglong  PTAL

File: docs/nightly/variates.yml [hunk 2]
```diff
@@ -1 +1,7 @@
 grafana-version: 9.5.15
+greptimedb-version: 0.7.2
+vector-version: 0.33.0-debian
+prometheus-version: 2.52.0
+node-exporter-version: 1.8.0
+go-sdk-version: 0.5.0
```
Comment (nicecui) [inline docs/nightly/variates.yml:6]: @yuanbohan PTAL

File: docs/nightly/variates.yml [hunk 3]
```diff
@@ -1 +1,7 @@
...
+java-sdk-version: 0.7.3
```
Comment (nicecui) [inline docs/nightly/variates.yml:7]: @fengjiachun PTAL

Comment (fengjiachun) [reply docs/nightly/variates.yml:7]: LGTM

**Review by sunchanglong — APPROVED** [review]

File: docs/nightly/variates.yml (hunk 2 above)
Comment (yuanbohan) [reply docs/nightly/variates.yml:6]: LGTM

**Review by yuanbohan — APPROVED** [review]

**Pull request merged**